README.md

```diff
@@ -244,11 +249,22 @@ end
 As a scientist, you know it's always important to be able to turn your experiment off, lest it run amok and result in villagers with pitchforks on your doorstep. In order to control whether or not an experiment is enabled, you must include the `enabled?` method in your `Scientist::Experiment` implementation.
 
 ```ruby
-class MyExperiment < ActiveRecord::Base
+class MyExperiment
```

Comment:
I like how the old example shows that `percent_enabled` can be persisted in a DB, but this is cleaner.